design/arp-load-sharing.md

```diff
@@ -0,0 +1,41 @@
+# ARP Load Sharing
+
+## Summary
+
+MetalLB in layer 2 mode does not support load balancing. See [here](https://metallb.universe.tf/concepts/layer2/).
```

Comment:
Well, load balancing is done by the Service load balancer once it hits a Kubernetes node.  I'd say that the load balancing is limited by going through a single node before being load balanced across the cluster.

Reply:
Right, I mean network load-balancing, not the compute side of it.
I can clarify.